python/cuml/test/test_fil.py

```diff
@@ -539,28 +549,31 @@ def test_lightgbm(tmp_path, num_classes, n_categorical):
                          random_state=43210,
                          classification=True)
     if n_categorical > 0:
-        X_fit = to_categorical(X, n_categorical)
+        X_fit, X = to_categorical(X, n_categorical)
     else:
         X_fit = X
 
     train_data = lgb.Dataset(X_fit, label=y)
     num_round = 5
     model_path = str(os.path.join(tmp_path, 'lgb.model'))
 
+    def fil_load():
+        return ForestInference.load(model_path,
+                                    algo='TREE_REORG',
+                                    output_class=True,
+                                    model_type="lightgbm")
+
```

Comment:
this was useful in debugging, and seems more readable to me

Reply:
I'm not sure if it's more readable. It removes the duplicate code, but also introduces an extra level of indirection, and a locally defined function. I think the current version is better.